docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       - '5156:5156' # mockpass ports
       - '9229:9229' # Node debugger port
     environment:
+      - NODE_OPTIONS=--max-old-space-size=2048
```

Comment:
I think this belongs in `Dockerfile.development` rather than `docker-compose`. `docker-compose` is for application logic, but this option is below the application layer.

Reply:
noted tks